### DISCUSSION TITLE
Installation no longer works

### DISCUSSION THREAD

**Discussion by edtwodth** [mKeRix/room-assistant#1368] [General] [answered]
Hi, I'm following the guide to install room-assistant on Raspberry Pi Zero W. But every time I try, the installation og room-assistant stop:

`npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'room-assistant@2.20.0',
npm WARN EBADENGINE   required: { node: '>= 12 < 17' },
npm WARN EBADENGINE   current: { node: 'v18.14.1', npm: '9.3.1' }
npm WARN EBADENGINE }
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated tar@2.2.2: This version of tar is no longer supported, and will not receive security updates. Please upgrade asap.
(##################) ⠏ reify:mathjs: http fetch GET 200 https://registry.npmjs.org/mathjs/-/mathjs-10.6.4.tgz 247793ms (cache miss)
`

**Comment by edtwodth** [comment, marked as answer]
Got it up and running, just required a lot of patience

**Reply by bedr0** [reply]
Hi @edtwodth, did it worked after a long time stuck at reify:mathjs or you had to restart the whole process and eventually it worked? 
Asking because I'm almost an hour into "sudo npm i --global --unsafe-perm room-assistant" and the script seems to be stuck at this part.

PS: It took a long time but has finished with "added 324 packages in 48m"
